### PR TITLE
Adds a global spores limit to blob factories

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Adds a global spore limit of 25 to blob factories
Increases the max spores from a single factory from 3 to 5

## Why It's Good For The Game
Blob spores are very potent in large amounts, therefore this global limit. Without the max spore increase, you'd start hitting this limit at ~8 factories, which is very late, and still offers a recharge of 8 spores per 10 seconds. By increasing the max spores per factory, the incentive is that you'll keep 5 "core" factories, after which their global effectiveness will die down and you'll only gain faster spore recharges

## Testing
Spawned in as blob, made factory, saw 5 spores come out of it

## Changelog
:cl:
tweak: Blobs now cannot sustain more than 25 spores globally
tweak: Blob factories can now sustain 5 spores instead of 3
/:cl:

### CODE REVIEW

File: code/modules/events/blob/blob_structures/factory.dm [hunk 1]
```diff
@@ -1,11 +1,13 @@
+GLOBAL_VAR_INIT(spores_active, 0)
```
Comment (Henri215) [inline code/modules/events/blob/blob_structures/factory.dm:1]: Is it something that can be scoped to SSmobs?

**Comment by Carthusia** [comment]
Is this per blob or a literal global count including all cores?

**Comment by DGamerL** [comment]
> This pull request seems to be stale as there have been no changes in 14 days, please make changes within 7 days or the PR will be closed. If you believe this is a mistake, please inform a development team member on Discord.

>:(

File: code/modules/events/blob/blob_structures/factory.dm (hunk 1 above)
Comment (DGamerL) [reply code/modules/events/blob/blob_structures/factory.dm:1]: Resolved on discord

**Review by Burzah — APPROVED** [review]

**Comment by Dafrek** [comment]
WIth how strong blob already is, i dont think it needed a buff

**Pull request merged**